.github/workflows/push-to-prod-old-fleets-nebraos-cloud.yml

```diff
@@ -0,0 +1,185 @@
+        with:
+          ref: production
+      - name: Export short SHA of current commit
+        shell: bash
+        run: |
+          FIRMWARE_SHORT_HASH=$( echo ${GITHUB_SHA:0:7} )
+          echo "FIRMWARE_SHORT_HASH=$FIRMWARE_SHORT_HASH" >> $GITHUB_ENV
+          
+          # Get the latest GA release and branch
+          source <(grep FIRMWARE_VERSION settings.ini)
+          LATEST_GA=$FIRMWARE_VERSION
+          GITHUB_BRANCH=$( echo "${{ github.ref }}" | sed 's/refs\/heads\///g' )
+          
+          echo "LATEST_GA=$LATEST_GA" >> $GITHUB_ENV
+          echo "GITHUB_BRANCH=$GITHUB_BRANCH" >> $GITHUB_ENV
+      - name: Random sleep (balena API workaround)
+        run:  sleep $[ ( $RANDOM % 540 ) + 1 ]s
+      - name: Generate Docker Compose for ROCK Pi
+        if: matrix.sbc == 'rockpi'
+        run: |
+          python gen_docker_compose.py rockpi -o docker-compose.yml
+          ROCKPI="-rockpi"
+          echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
+          cp balena/balena.yml.${{ matrix.sbc }}-${{ matrix.variant }} balena.yml
+      - name: Generate Docker Compose for Nebra RasPi
+        if: matrix.sbc == 'raspi'
+        run: |
+          python gen_docker_compose.py rpi -o docker-compose.yml
+          cp balena/balena.yml.${{ matrix.sbc }}-${{ matrix.variant }} balena.yml
+      - name: Generate Docker Compose for RAK and Non-Nebra RasPi
+        if: matrix.sbc == 'rak' || matrix.sbc == 'og' || matrix.sbc == 'sensecap' || matrix.sbc == 'finestra' || matrix.sbc == 'controllino' || matrix.sbc == 'cotx' || matrix.sbc == 'pantherx1' || matrix.sbc == 'linxdot' || matrix.sbc == 'syncrobit' || matrix.sbc == 'risinghf' || matrix.sbc == 'midas'
+        run: |
+          python gen_docker_compose.py rpi -o docker-compose.yml
+          cp balena/balena.yml.${{ matrix.sbc }} balena.yml
+      - name: Generate Docker Compose for Non-Nebra ROCK Pi
+        if: matrix.sbc == 'linxdot-rkcm3' || matrix.sbc == 'syncrobit-rkcm3'
+        run: |
+          python gen_docker_compose.py rockpi -o docker-compose.yml
+          ROCKPI="-rockpi"
+          echo "ROCKPI=$ROCKPI" >> $GITHUB_ENV
+          cp balena/balena.yml.${{ matrix.sbc }} balena.yml
+      - name: Generate Docker Compose for Pycom
+        if: matrix.sbc == 'pycom'
+        run: |
+          python gen_docker_compose.py pycom -o docker-compose.yml
+          cp balena/balena.yml.pycom balena.yml
+      - name: Generate Docker Compose for Pisces
+        if: matrix.sbc == 'pisces'
+        run: |
+          python gen_docker_compose.py pisces -o docker-compose.yml
+          cp balena/balena.yml.pisces balena.yml
+      - name: Balena Deploy (Nebra ROCK Pi)
+        uses: nebraltd/balena-cli-action@v15.0.3
+        if: matrix.sbc == 'rockpi' && success()
+        with:
+          balena_api_token: ${{secrets.NEBRAOS_CLOUD_TOKEN}}
+          balena_command: "deploy nebraltd/helium-${{ matrix.variant }}-${{ matrix.frequency }}${{ env.ROCKPI }} --debug --nocache --build"
```

Comment:
You need `nebra_cloud: true` for all of the deployment steps

Reply:
yeah, forgot.